pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -8,50 +11,59 @@
 
 class TestPaverBokChoyCmd(unittest.TestCase):
 
-    def setUp(self):
-        self.request = BokChoyTestSuite('')
-
-    def _expected_command(self, expected_text_append, expected_default_store=None):
-        if expected_text_append:
-            expected_text_append = "/" + expected_text_append
+    def _expected_command(self, name, store=None):
         expected_statement = (
             "DEFAULT_STORE={default_store} "
             "SCREENSHOT_DIR='{repo_dir}/test_root/log' "
             "BOK_CHOY_HAR_DIR='{repo_dir}/test_root/log/hars' "
             "SELENIUM_DRIVER_LOG_DIR='{repo_dir}/test_root/log' "
-            "nosetests {repo_dir}/common/test/acceptance/tests{exp_text} "
+            "nosetests {repo_dir}/common/test/acceptance/{exp_text} "
             "--with-xunit "
             "--xunit-file={repo_dir}/reports/bok_choy/xunit.xml "
             "--verbosity=2 "
         ).format(
-            default_store=expected_default_store,
+            default_store=store,
             repo_dir=REPO_DIR,
-            exp_text=expected_text_append,
+            exp_text=name,
         )
         return expected_statement
 
-    def test_default_bokchoy(self):
-        self.assertEqual(self.request.cmd, self._expected_command(''))
+    def test_default(self):
+        self.suite = BokChoyTestSuite('')
```

Comment:
This looks like it could just be `suite` instead of `self.suite` since it is only every used within this function.